data/service/data-service.js

```diff
@@ -1332,6 +1407,7 @@ exports.DataService = Montage.specialize(/** @lends DataService.prototype */ {
                         return mapping.mapRawDataToObjectProperty(data, object, propertyName);
                     });
                 } else {
+                    //This was already done a few lines up. Why are we re-doing this?
```

Comment:
I agree, this seems redundant.

Reply:
ok, filed https://github.com/montagejs/montage/issues/2013